### DISCUSSION TITLE
Open lsp_document_symbols from selected entry inside files/buffer picker

### DISCUSSION THREAD

**Discussion by egemenkopuz** [ibhagwan/fzf-lua#1530] [Q&A]
Hi all,

I just started using fzf-lua, and its been amazing experience so far. I was wondering if any of you have came up with a proper way to open pickers like lsp_document_symbols or diagnostics_document from selected entry inside files and buffer pickers via keybinding.

**Comment by ibhagwan** [comment]
For diagnostics, use the actions table to define a new key bind that calls document/workspace diagnostics/symbols.

Read the wiki and search the issues, there are loads of examples how to add a custom keybind.
